validator/keymanager/local/import.go

```diff
@@ -17,6 +16,13 @@ import (
 )
 
 // ImportKeystores into the local keymanager from an external source.
+// protection history in the database.
```

Comment:
Whats this?

Reply:
hmm i wonder where that came from will remove it.